test/alternator/suite.yaml

```diff
@@ -1 +1,23 @@
-type: Run
+type: Python
+pool_size: 4
+extra_scylla_cmdline_options:
+#   - '--alternator-address=8080'
+    - '--alternator-enforce-authorization=1'
+    - '--alternator-write-isolation=always_use_lwt'
+    - '--alternator-streams-time-window-s=0'
+    - '--alternator-timeout-in-ms=30000'
+    - '--alternator-ttl-period-in-seconds=0.5'
+#    - '--alternator_port=8000'
+  # Allow testing experimental features. Following issue #9467, we need
+  # to add here specific experimental features as they are introduced.
+  # We only list here Alternator-specific experimental features - CQL
+  # ones are listed in test/cql-pytest/run.py.
+#    - '--experimental-fa   eatures=alternator-streams'
+run_first:
+    - test_streams
+    - test_scan
+    - test_tracing
+    - test_ttl
+extra_scylla_config_options:
+  { alternator_port: 8000,
+    alternator_write_isolation: only_rmw_uses_lwt}
```

Comment:
i this all that we need to do to adopt alternator tests to test.py is add additional option here

prepare_cmd: "INSERT INTO system_auth_v2.roles (role, salted_hash) VALUES ('alternator', 'secret_pass')"

and perform this command in test.py here https://github.com/scylladb/scylladb/blob/2cf7cc1ea558e72c6a173c8ceffafcf93b426f4f/test.py#L979
 after ```cluster.before_test(self.uname)```